packages/x-charts-pro/tsconfig.build.json

```diff
@@ -8,8 +8,7 @@
     "noEmit": false,
     "emitDeclarationOnly": true,
     "outDir": "build/esm",
-    "rootDir": "./src",
-    "types": ["node", "@mui/material/themeCssVarsAugmentation"]
```

Comment:
This is a leftover after experimentation, but I'm not sure they were necessary, the `./tsconfig.json` this file extends already has these and some other types defined... 🤷 🤔